postgres/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:16.04
+ENV RUNTIME_PACKAGES="python3"
+ENV BUILD_PACKAGES="curl build-essential python3-dev ca-certificates libssl-dev libffi-dev postgresql libpq-dev"
+
+RUN apt-get update && apt-get install -y $RUNTIME_PACKAGES $BUILD_PACKAGES && curl -sS https://bootstrap.pypa.io/get-pip.py | python3
```

Comment:
Copied from: https://github.com/ONSdigital/sdx-sequence/pull/10

Python 3.5.2.

It wasn't installed (which did surprise me a little), hence why I added it. It's similar to what they did in the flask-crypto one, so maybe its missing from the Ubuntu image?